### PR TITLE
recoding added

### PR DESCRIPTION
**Pull request by msidqi**
recoding module

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -55,6 +55,13 @@ const module_mgheber = (msg) => {
 	}
 }
 
+const module_recoding = (msg) => {
+	const recode = /tla3.*(dir)*.*[R|r]ecoding[?]*|[R|r]ecoding[ ]*\?/g
```
Comment (AlaaZorkane) [inline index.js]: It's better to take msg.content.toLowerCase() as reference instead of the regex

File: index.js [hunk 2]
```diff
@@ -55,6 +55,13 @@ const module_mgheber = (msg) => {
...
+	if (msg.content.match(recode)) {
+		msg.channel.send ('La');
```
Comment (AlaaZorkane) [inline index.js]: `msg.channel.send('La')` instead of `msg.channel.send ('La')`

Comment (msidqi) [reply index.js]: both work ya zrg

Comment (mehdibo) [reply index.js]: It's about consistency al mghebber

**Pull request merged**